mathesar/tests/api/conftest.py

```diff
@@ -23,31 +21,6 @@ def client():
     return APIClient()
 
 
-@pytest.fixture
-def create_schema(engine, test_db_model):
```

Comment:
In 733f4bfc927c4f609a23f84038ef7fa0fdafb972 I moved `create_schema` and `create_table` up so that I could use them in my own `go_to_patents_data_table` fixture that I added in 2e06b25d7565b8e6cda1e9c06812b748ced3568b. My understanding is that I can only access fixtures defined within ancestor directories -- not within sibling/cousin directories. Correct?